.github/workflows/publish_main_docker_image.yaml

```diff
@@ -0,0 +1,32 @@
+name: Publish Docker image to Docker Hub (main)
+
+on:
+  push:
+    branches: [main]
```

Comment:
I'd rather have a manual trigger. What do you think?

Reply:
I like having an automatic build for the `main` branch. That way you know that `quickwit/quickwit:main` always reflects the latest version of `main`. Having a manual trigger for another tag makes sense though. I don't think we'd want our deployments to point to main.